### DISCUSSION TITLE
History for Advanced Search

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6113] [Ideas]
While `$:/AdvancedSearch` allows me to create customized search filter and save them, but it would be quite handy to keep a small history of the recently entered search filter (e.g. write filter and press enter or click search button, etc. ) and when one returns to `$:/AdvancedSearch` page can simply use the up/down arrow keys to bring them back

What do you think?

**Comment by pmario** [comment]
Have a look at my [Advanced Search Plus Plugin](https://wikilabs.github.io/editions/advanced-search-plus/) it allows you to save "searches". It's intended for power-users, because I needed to strip out the default tab-text to make place for saved configs

**Reply by dubiouscript** [reply]
:+1: 
https://github.com/Jermolene/TiddlyWiki5/issues/4475#issuecomment-618371300
>some kind of
~/.tw_search_history
would be a
nice to have
addition

**Comment by Jermolene** [comment]
Hi @kookma it would definitely be useful in some situations. From the core perspective, I think it would have to be off by default because search history can be quite personal data, and we'd want to avoid people exposing it unwittingly.
